vaadin-ordered-layout-flow/src/main/java/com/vaadin/flow/component/orderedlayout/FlexLayout.java

```diff
@@ -98,14 +98,28 @@ public FlexLayout(Component... children) {
     }
 
     /**
+     * @deprecated Rename setWrapMode to setFlexWrap
      * Gets the {@link WrapMode} used by this layout.
      * <p>
      * The default flex wrap mode is {@link WrapMode#NOWRAP}.
      *
      * @param wrapMode the flex wrap mode of the layout, never
      *                     <code>null</code>
      */
+    @Deprecated
     public void setWrapMode(WrapMode wrapMode) {
+        this.setFlexWrap(wrapMode);
+    }
+
+    /**
+     * Gets the {@link WrapMode} used by this layout.
+     * <p>
+     * The default flex wrap mode is {@link WrapMode#NOWRAP}.
+     *
+     * @param wrapMode the flex wrap mode of the layout, never
+     *                     <code>null</code>
+     */
+    public void setFlexWrap(WrapMode wrapMode) {
```

Comment:
Also deprecate the `WrapMode` enum and create a new one (`FlexWrap`)

Reply:
thank you for comment. It is done.